build_projects/dotnet-cli-build/CompileTargets.cs

```diff
@@ -418,11 +279,23 @@ private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli d
             }
 
             // cleanup compilers project output we don't need
-            DeleteMainPublishOutput(outputDir, "compilers");
+            PublishMutationUtilties.CleanPublishOutput(outputDir, "compilers");
             File.Delete(compilersDeps);
-            File.Delete(compilersRuntimeConfig);
+            
+            // Publish SharedFx
+            var sharedFrameworkNugetVersion = c.BuildContext.Get<string>("SharedFrameworkNugetVersion");
+            var commitHash = c.BuildContext.Get<string>("CommitHash");
+
+            var sharedFrameworkPublisher = new SharedFrameworkPublisher(
```

Comment:
Shouldn't it be published once in `dotnet-host-build`, and then `dotnet-cli-build` take what was published during `dotnet-host-build` and use it?

I'm thinking when these are split repos:

- `host-build` publishes the shared fx
- `host-build` creates the .zip and publishes it to Azure
- `cli-build` pulls the .zip, and extracts it locally

Or were you thinking it would work differently?